src/script/api.ts

```diff
@@ -0,0 +1,37 @@
+const API_URL = 'https://dummyjson.com/posts';
+const LIMIT = 30;
+let skip = 0;
+let hasMore = true;
+let isFetching = false;
+
+export interface Post {
+    id: number;
+    title: string;
+    body: string;
+}
+
+// Fetch posts from API
+export const fetchPosts = async (): Promise<Post[]> => {
```

Comment:
can we use `Array<Post>`.

```suggestion
export const fetchPosts = async (): Promise<Array<post>> => {
```

Reply:
change https://github.com/Sakshi-Simform/product-management/commit/0ad89dad04323d8da0a14f856deb28e24eef7889